.github/workflows/daily-sanitizers.yml

```diff
@@ -0,0 +1,92 @@
+name: daily-sanitizers
+
+on:
+  schedule:
+    - cron: '0 4 * * *' # run at 4 AM UTC
+
+jobs:
+  build:
+    runs-on: [self-hosted, linux, ARM64]
+    strategy:
+      matrix:
+        # Test of these containers
+        container: ["ubuntu-dev:22"]
```

Comment:
so if both tests fail also on ubuntu 22.04 why do we run them on this ubuntu verstion and not 20.04
I thought that running them on ubuntu 22.04 will make them pass

Reply:
I am fine running whatever ubuntu version we need for them to pass if such exists. we can still release via ubuntu 20